### PR TITLE
subnets-documentation

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Subnets documentation

### CODE REVIEW

File: docs/how-to-guides/how-to-enable-event-observer-interface.md
Comment (kantai) [inline docs/how-to-guides/how-to-enable-event-observer-interface.md:1]: I don't think this file should be changed at all here.

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,892 +1,53 @@
...
-In the subnet, this asset should not be owned by anyone (`none`):
-
-```sh
-node ./verify.js 2
-```
+- [Create contracts](how-to-guides/how-to-create-contracts.md)
+- [Start devnet](how-to-guides/how-to-start-devnet.md)
+- [Set up node.js scripts](how-to-guides/how-to-setup-nodejs-scripts.md)
+- [Interact with subnets](how-to-guides/how-to-interact-with-subnet.md)
+- [Enable event observer interface](how-to-guides/how-to-enable-event-observer-interface.md)
```
Comment (kantai) [inline docs/getting-started.md:53]: Splitting up these items into different articles still isn't making sense to me. Have you tried using the commands from these articles independently of one another? I don't think they would work.

File: docs/getting-started.md [hunk 2]
```diff
@@ -23,21 +23,19 @@ There are two ways to test your applications on a subnet. Running a local subnet
...
 
-Make sure you have [`clarinet`](https://github.com/hirosystems/clarinet/releases/tag/v1.5.3) installed and the **clarinet version is at 1.5.3 or above**. If you do not already have Clarinet installed, you can refer to the instructions [here](https://docs.hiro.so/smart-contracts/clarinet#installing-clarinet) for installation procedures.
+Make sure you have [`clarinet`](https://github.com/hirosystems/clarinet/releases/tag/v1.5.3) installed and the **clarinet version is at 1.5.3 or above**.
+If you have clarinet installed but you want to know the version, you can navigate to your terminal and type `clarinet`. You'll see a version of the clarinet. If you have an older version of the Clarinet and need an upgrade to a specific version, you can use the following command in your terminal:
+`brew install clarinet@1.5.3`.
+If you do not already have Clarinet installed, you can refer to the instructions [here](https://docs.hiro.so/clarinet/getting-started#install-clarinet) for installation procedures.
 
-To create a new project, run:
+You can create a new project by running the following command or by using [Hiro platform](https://docs.hiro.so/platform/create-project).
```
[the remark this answers is not in the document]
Comment (LakshmiLavanyaKasturi) [reply docs/getting-started.md:38]: Not the subnet yet, but creating or importing a project, and creating a contract can happen using the Hiro Platform.

File: docs/getting-started.md (hunk 1 above)
Comment (LakshmiLavanyaKasturi) [reply docs/getting-started.md:53]: I didn't mean to have these articles work independently. I still meant to have these articles following sequentially to interact with subnets. But I understand that splitting these articles might confuse users when they jump between them and end up nowhere.

File: docs/how-to-guides/how-to-enable-event-observer-interface.md
Comment (kantai) [inline docs/how-to-guides/how-to-enable-event-observer-interface.md:1]: This file shouldn't be renamed.

**Review by kantai — APPROVED** [review]

**Pull request merged**